tools/golangci-override.yaml

```diff
@@ -40,6 +40,12 @@ linters-settings:
             desc: "do not use gopkg.in/yaml.v3; use sigs.k8s.io/yaml instead"
           - pkg: github.com/ghodss/yaml
             desc: "do not use github.com/ghodss/yaml; use sigs.k8s.io/yaml instead"
+          - pkg: github.com/envoyproxy/go-control-plane/envoy/config/cluster/v3
```

Comment:
I think I'm using the wrong package scope here, as the overrides I added don't seem to be being applied during a local `make lint`

Reply:
This was totally wrong under `depguard`, but I couldn't get exclude rules or `staticcheck` config working properly either.